src/main/java/no/nav/tag/tiltaksgjennomforingprosess/JournalpostJobb.java

```diff
@@ -56,15 +57,23 @@ public void avtalerTilJournalfoering() {
     }
 
     private Map<UUID, String> journalfoerAvtaler(String stsToken, List<Avtale> avtalerTilJournalforing) {
-        return avtalerTilJournalforing
-                .parallelStream()
-                .map(avtale -> {
-                    avtale.setJournalpostId(
-                            joarkService.sendJournalpost(stsToken, journalpostFactory.konverterTilJournalpost(avtale))
-                    );
-                    return avtale;
-                })
-                .collect(Collectors.toMap(Avtale::getId, Avtale::getJournalpostId));
+
+        Map<UUID, String> journalfoerteAvtaler = new HashMap<>(avtalerTilJournalforing.size());
+        avtalerTilJournalforing
```

Comment:
I stedet for `avtalerTilJournalforing.forEach` og `.put` kan man bruke `avtalerTilJournalforing.map` i stedet